app/views/welcome/_available_games.html.erb

```diff
@@ -1,12 +1,13 @@
 <% if @games.present? %>
   <% @games.each do |game| %>
-    <% if game.white_user.nil? || game.black_user.nil? %>
+    <% if game.white_user_id.nil? || game.black_user_id.nil? %>
       <h2>
+        <%= link_to 'Join Game', game_path(game, id: game.id), method: :patch, :class => 'btn btn-success btn-large join-game-link' %>
```

Comment:
this sends the game id of the game the user wants to join to the game controller update action